### PR TITLE
Initial Generation of data catalog

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: google-api-grpc/pom.xml [hunk 1]
```diff
@@ -777,7 +777,7 @@
     <module>grpc-google-cloud-bigtable-v2</module>
     <module>grpc-google-cloud-container-v1</module>
     <module>grpc-google-cloud-containeranalysis-v1beta1</module>
-    <module>grpc-google-cloud-datalabeling-v1beta1</module>
+    <module>grpc-google-cloud-datacatalog-v1beta1</module><module>grpc-google-cloud-datalabeling-v1beta1</module>
```
Comment (kolea2) [inline google-api-grpc/pom.xml]: formatting

File: google-api-grpc/pom.xml [hunk 2]
```diff
@@ -840,7 +840,7 @@
     <module>proto-google-cloud-bigtable-v2</module>
     <module>proto-google-cloud-container-v1</module>
     <module>proto-google-cloud-containeranalysis-v1beta1</module>
-    <module>proto-google-cloud-datalabeling-v1beta1</module>
+    <module>proto-google-cloud-datacatalog-v1beta1</module><module>proto-google-cloud-datalabeling-v1beta1</module>
```
Comment (kolea2) [inline google-api-grpc/pom.xml]: formatting

File: google-cloud-clients/pom.xml [hunk 3]
```diff
@@ -787,7 +787,7 @@
...
...center.v1.stub:com.google.cloud.securitycenter.v1beta1.stub:com.google.cloud.spanner-admin-database.v1.stub:com.google.cloud.spanner-admin-instance.v1.stub:com.google.cloud.spanner.v1.stub:com.google.cloud.speech.v1.stub:com.google.cloud.speech.v1beta1.stub:com.google.cloud.speech.v1p1beta1.stub:com.google.cloud.talent.v4beta1.stub:com.google.cloud.tasks.v2.stub:com.google.cloud.tasks.v2beta2.stub:com.google.cloud.tasks.v2beta3.stub:com.google.cloud.texttospeech.v1.stub:com.google.cloud.texttospeech.v1beta1.stub:com.google.cloud.trace.v1.stub:com.google.cloud.trace.v2.stub:com.google.cloud.translate.v3beta1.stub:com.google.cloud.video-intelligence.v1.stub:com.google.cloud.video-intelligence.v1beta1.stub:com.google.cloud.video-intelligence.v1beta2.stub:com.google.cloud.video-intelligence.v1p1beta1.stub:com.google.cloud.video-intelligence.v1p2beta1.stub:com.google.cloud.video-intelligence.v1p3beta1.stub:com.google.cloud.vision.v1.stub:com.google.cloud.vision.v1p1beta1.stub:com.google.cloud.vision.v1p2beta1.stub:com.google.cloud.vision.v1p3beta1.stub:com.google.cloud.vision.v1p4beta1.stub:com.google.cloud.webrisk.v1beta1.stub:com.google.cloud.websecurityscanner.v1alpha.stub</packages>
```
Comment (kolea2) [inline google-cloud-clients/pom.xml]: add to modules

File: google-cloud-bom/pom.xml [hunk 4]
```diff
@@ -1045,7 +1045,7 @@
         <artifactId>google-cloud-webrisk</artifactId>
         <version>0.2.1-alpha-SNAPSHOT</version><!--{x-version-update:google-cloud-webrisk:current}-->
       </dependency>
-    </dependencies>
+    <dependency><groupId>com.google.api.grpc</groupId><artifactId>proto-google-cloud-datacatalog-v1beta1</artifactId><version>0.0.1-alpha-SNAPSHOT</version><!--{x-version-update:proto-google-cloud-datacatalog-v1beta1:current}--></dependency><dependency><groupId>com.google.api.grpc</groupId><artifactId>grpc-google-cloud-datacatalog-v1beta1</artifactId><version>0.0.1-alpha-SNAPSHOT</version><!--{x-version-update:grpc-google-cloud-datacatalog-v1beta1:current}--></dependency><dependency><groupId>com.google.cloud</groupId><artifactId>google-cloud-datacatalog</artifactId><version>0.0.1-alpha-SNAPSHOT</version><!--{x-version-update:google-cloud-datacatalog:current}--></dependency></dependencies>
```
Comment (kolea2) [inline google-cloud-bom/pom.xml]: formatting

**Comment by sduskis** [comment]
@kolea2, this should be ready to go.

**Pull request merged**